player-page.html

```diff
@@ -25,7 +25,7 @@
             
```

Comment:
please add hover to your navbar button
kindly use the correct image for the page